externs/shaka/player.js

```diff
@@ -768,6 +769,13 @@ shaka.extern.DashManifestConfiguration;
  *   <code>EXT-X-PROGRAM-DATE-TIME</code> tags in the manifest.
  *   Meant for tags that are incorrect or malformed.
  *   <i>Defaults to <code>false</code>.</i>
+ * @property {string} mediaPlaylistFullMimeType
+ *   A string containing a full mime type, including both the basic mime type
+ *   and also the codecs. Used when the HLS parser parses a media playlist
+ *   directly.
+ *   You can use the <code>shaka.util.MimeUtils.getFullType()</code> utility to
+ *   format this value.
+ *   <i>Defaults to <code>'video/mp4; codecs="avc1.42E01E"'</code>.</i>
```

Comment:
I think we should assume muxed content with AAC audio by default.  (Add ", mp4a.40.2")

Reply:
most of the content is TS, I think that the default should be TS instead of MP4